catalog/rest/catalog-rest-service/pom.xml

```diff
@@ -54,10 +133,14 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Import-Package>
-                            !org.codice.ddf.attachment,
-                            *
-                        </Import-Package>
+                        <Embed-Dependency>
+                            asm,
+                            catalog-core-actions,
+                            catalog-core-api-impl,
+                            json-smart,
+                            platform-util,
+                            owasp-java-html-sanitizer
+                        </Embed-Dependency>
                     </instructions>
```

Comment:
✏️ Not absolutely necessary, But I think it communicates it a lot better what this bundle is providing if we explicitly list the `Export-Package`